x-pack/plugins/observability_solution/inventory/common/entities.ts

```diff
@@ -86,16 +89,19 @@ interface ServiceEntity extends BaseEntity {
   [ENTITY_TYPE]: 'service';
   [SERVICE_NAME]: string;
   [SERVICE_ENVIRONMENT]?: string | null;
+  [AGENT_NAME]: AgentName | AgentName[] | null;
```

Comment:
Let's not use `AgentName` here as we discussed today an agent name can be pretty much anything from apm agent to the user's host name.